src/impl_views.rs

```diff
@@ -137,6 +137,9 @@ where
 
     /// Return the array’s data as a slice, if it is contiguous and in standard order.
     /// Return `None` otherwise.
+    // TODO: Should this be `to_slice` if taking a reference?
```

Comment:
Yes, I agree `to_slice` is a better name. (See also #262.) Will you please create a separate PR for this?